apps/page/Dockerfile

```
@@ -0,0 +1,24 @@
+RUN npm install -g pnpm
+
+RUN pnpm install --frozen-lockfile
+
+COPY . .
+
+WORKDIR /app/apps/my-frontend
+RUN pnpm run build # Or whatever your build command is
+
+EXPOSE 3000
+
+CMD ["pnpm", "start"] # Or "node dist/index.js", etc.
```

Hadolint (2.12.0)

[error] unexpected '#'
expecting '\', a new line followed by the next instruction, or at least one space

(DL1000)